app/src/main/java/com/sschuraytz/charcoaldrawing/CharcoalTool.java

```diff
@@ -0,0 +1,89 @@
+package com.sschuraytz.charcoaldrawing;
+
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.graphics.PorterDuff;
+
+import java.util.Random;
+
+
+public class CharcoalTool{
+
+    private Canvas canvas;
+    private Bitmap bitmap;
+    private Paint paint;
+    private Random rand = new Random();
+    private int radius;
+
+    public CharcoalTool() {
+        bitmap = Bitmap.createBitmap(200, 200, Bitmap.Config.ARGB_8888);
+        canvas = new Canvas(bitmap);
+        paint = new Paint(Paint.ANTI_ALIAS_FLAG);
+        //should allow user to modify
+        paint.setStrokeWidth(3);
+        radius = 35;
+        createCircleWithNoLocation();
+    }
+
+    private void createCircleWithNoLocation()
```

Comment:
It's weird to have a method, with no arguments, just call another method with no arguments.